src/engine-actions/get-max-channel.js

```diff
@@ -8,16 +8,42 @@ const { listChannels } = require('../lnd-actions')
  * @return {String} res.maxBalance - the max balance in all open channels.
  */
 async function getMaxChannel ({ outbound = true } = {}) {
-  const { channels = [] } = await listChannels({ client: this.client })
+  const [
+    { channels = [] } = {},
+    { pendingOpenChannels = [] } = {}
+  ] = await Promise.all([
+    // The response from listChannels consists of channels that may be active or inactive
+    listChannels({ client: this.client }),
+    listPendingChannels({ client: this.client })
+  ])
   const balance = outbound ? 'localBalance' : 'remoteBalance'
 
-  if (channels.length === 0) {
-    this.logger.debug('getMaxChannel: No channels exist')
+  if (channels.length === 0 && pendingOpenChannels.length === 0) {
+    this.logger.debug('getMaxChannel: No open or pending channels exist')
     return {}
   }
 
-  const maxBalance = channels.reduce((max, channel) => Big(channel[balance]).gt(max) ? Big(channel[balance]) : max, Big(channels[0][balance]))
-  return { maxBalance: maxBalance.toString() }
+  const maxBalance = channels.reduce((max, channel) => {
+    if (Big(channel[balance]).gt(max)) {
```

Comment:
[nit] not apart of your PR, but using `balanceType` instead of `balance` would make this code more readable